processing/src/main/java/io/druid/query/ordering/StringComparators.java

```diff
@@ -34,23 +34,23 @@
 {
   public static final String LEXICOGRAPHIC_NAME = "lexicographic";
   public static final String ALPHANUMERIC_NAME = "alphanumeric";
-  
+
   public static final LexicographicComparator LEXICOGRAPHIC = new LexicographicComparator();
   public static final AlphanumericComparator ALPHANUMERIC = new AlphanumericComparator();
-    
+
   @JsonTypeInfo(use=Id.NAME, include=As.PROPERTY, property="type", defaultImpl = LexicographicComparator.class)
   @JsonSubTypes(value = {
       @JsonSubTypes.Type(name = StringComparators.LEXICOGRAPHIC_NAME, value = LexicographicComparator.class),
       @JsonSubTypes.Type(name = StringComparators.ALPHANUMERIC_NAME, value = AlphanumericComparator.class)
   })
-  public static interface StringComparator extends Comparator<String>
+  public static interface StringComparator extends Comparator<Comparable>
   {
   }
-  
+
   public static class LexicographicComparator implements StringComparator
```

Comment:
Can we call this "ToStringLexicographicComparator" instead to reflect the fact that it will do `toString` on the objects?